### ISSUE TITLE
Exeception while resizing terminal window. 3.0.1

### ISSUE THREAD

**Issue by zella** [mabe02/lanterna#385]
When I resize window by dragging from bottom to top i gets error:

> Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 0
> 	at com.googlecode.lanterna.graphics.BasicTextImage.copyTo(BasicTextImage.java:214)
> 	at com.googlecode.lanterna.screen.ScreenBuffer.copyFrom(ScreenBuffer.java:140)
> 	at com.googlecode.lanterna.screen.AbstractScreen$1.drawImage(AbstractScreen.java:133)
> 	at com.googlecode.lanterna.graphics.AbstractTextGraphics.drawImage(AbstractTextGraphics.java:215)
> 	at com.googlecode.lanterna.gui2.TextGUIGraphics.drawImage(TextGUIGraphics.java:237)
> 	at com.googlecode.lanterna.gui2.MultiWindowTextGUI.drawGUI(MultiWindowTextGUI.java:265)
> 	at com.googlecode.lanterna.gui2.AbstractTextGUI.updateScreen(AbstractTextGUI.java:120)
> 	at com.googlecode.lanterna.gui2.MultiWindowTextGUI.updateScreen(MultiWindowTextGUI.java:223)
> 	at com.googlecode.lanterna.gui2.AbstractTextGUIThread.processEventsAndUpdate(AbstractTextGUIThread.java:83)
> 	at com.googlecode.lanterna.gui2.MultiWindowTextGUI.waitForWindowToClose(MultiWindowTextGUI.java:409)
> 	at com.googlecode.lanterna.gui2.AbstractWindow.waitUntilClosed(AbstractWindow.java:269)
> 	at com.googlecode.lanterna.gui2.MultiWindowTextGUI.addWindowAndWait(MultiWindowTextGUI.java:373)
> 	at Issue.main(Issue.java:20)

Scenario:

Resize window by dragging from bottom to top:
![_2018-07-13_22-03-16](https://user-images.githubusercontent.com/3839393/42709358-f447b028-86e8-11e8-99a6-7f77fbc48086.png)

Code:
```
import com.googlecode.lanterna.gui2.*;
import com.googlecode.lanterna.screen.Screen;
import com.googlecode.lanterna.terminal.DefaultTerminalFactory;

import java.util.Collections;

public class Issue {
    public static void main(String[] args) throws Exception {

        DefaultTerminalFactory terminalFactory = new DefaultTerminalFactory();
        Screen screen = terminalFactory.createScreen();
        screen.startScreen();

        final WindowBasedTextGUI textGUI = new MultiWindowTextGUI(screen);

        final Window window = new BasicWindow("My Root Window");
        window.setHints(Collections.singletonList(Window.Hint.FULL_SCREEN));

        textGUI.addWindowAndWait(window);

    }
}
```

**Comment by avl42** [comment]
I was pretty sure that this was fixed for latest release. Before I look at
it more  closely, please ensure that you *really* use latest  version of
lanterna.

Andrey Zelyaev <notifications@github.com> schrieb am Fr., 13. Juli 2018
21:06:

**Comment by zella** [comment]
3.0.1 absolutely. Seems like window FULL_SCREEN problem

**Comment by avl42** [comment]
I apologize for my first answer.  It is indeed a different bug than the one
fixed in 3.0.1

I'll prepare a PR shortly...

**Comment by MartyIX** [comment]
@avl42 @mabe02 I guess the issue can be closed now, right?
